### PR TITLE
New connector : Honeywell Forge

### PR DESCRIPTION
**Pull request by ashishdotme**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/HoneywellForge/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "connectorId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify an ID for your connector. Connector ID is a unique identifier that represents your connector in Honeywell Forge.",
+            "x-ms-summary": "Connector ID",
+            "x-example": "e533b6fa-d7ee-4bed-bf30-02f433df86e6",
```
Comment (sriyen-msft) [inline certified-connectors/HoneywellForge/apiDefinition.swagger.json:146]: @ashishdotme Secret-look-a-like values are not allowed in any connector artifacts, please remove them. If you can not please provide business justification.

Applies same to similar values in the connector files

File: certified-connectors/HoneywellForge/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                    "value": "https://forge-sso.spec.honeywell.com/oauth2/token"
+                },
+                "refreshUrl": {
+                    "value": "https://forge-sso.spec.honeywell.com/oauth2/token"
+                }
+            }
+        }
+      }
+    },
+    "iconBrandColor": "#EE4677"
```
Comment (sriyen-msft) [inline certified-connectors/HoneywellForge/apiProperties.json:33]: @ashishdotme You need to add `"publisher"` and `"stackOwner"` properties as well.

Documentation - [Add metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4a-publisher-and-stackowner-properties)

File: certified-connectors/HoneywellForge/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,432 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Honeywell Forge",
+    "description": "Honeywell Forge is an Enterprise Performance Management Software-as-a-Service. It enables autonomous control of devices and codifies domain knowledge of assets in a contextual way. Use Honeywell Forge Connector to subscribe to Honeywell Forge to receive service cases and to close existing service cases.",
+    "version": "1.0",
+    "contact": {
+      "name": "Honeywell Support",
+      "url": "https://www.honeywell.com/us/en/contact/support"
```
Comment (sriyen-msft) [inline certified-connectors/HoneywellForge/apiDefinition.swagger.json:9]: @ashishdotme Can you please add `"email"` property as well.

Documentation - [Add metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4b-product-or-end-service-metadata)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@ashishdotme Please sign CLA to get the PR reviewed again

**Comment by ashishdotme** [comment]
@sriyen-msft We are abandoning this PR and raised a new PR with different account. We have addressed the review comments in that PR. Thanks!

**Pull request closed without merging**